cmd/wallet-web/test/specs/get.spec.js

```diff
@@ -147,11 +143,14 @@ describe.skip('sharing a credential from wallet - PresentationExchange', functio
   // mount vue component
   let wrapper;
   before(async function () {
+    const store = setup.getStateStore();
     wrapper = mount(Get, {
-      localVue,
-      store: setup.getStateStore(),
-      mocks: {
-        $webCredentialHandler: credHandler,
+      global: {
+        plugins: [store, i18n],
+        mocks: {
+          $webCredentialHandler: credHandler,
+          t: () => '',
```

Comment:
just curious this t  signifies test or what ??

Reply:
Since we are `vue-i18n` translations asynchronously from the `main.js` the `t` here cannot find them when we initialize the plugin here. So I just ended up mocking it